src/server/src/main/java/io/cassandrareaper/jmx/HostConnectionCounters.java

```diff
@@ -45,9 +45,9 @@ void incrementSuccessfulConnections(String host) {
       LOG.debug("Host {} has {} successfull connections", host, successes);
       if (null != metricRegistry) {
         metricRegistry
-            .counter(
-                MetricRegistry.name(
-                    JmxConnectionFactory.class, "connections", host.replace('.', '-')))
+            .counter(MetricRegistry.name(JmxConnectionFactory.class,
+                                         "connections",
+                                         host.replace('.', 'x').replaceAll("[^A-Za-z0-9]", "")))
```

Comment:
Can you undo the line wrapping changes. I believe the additional .replaceAll call doesn't break the line-length code style rule.
```suggestion
            .counter(
                MetricRegistry.name(
                    JmxConnectionFactory.class, "connections",host.replace('.', 'x').replaceAll("[^A-Za-z0-9]", "")))                                        
```